code/modules/mob/living/carbon/human/species/machine.dm

```diff
@@ -113,6 +113,28 @@
 			H.update_hair()
 			H.update_fhair()
 
+/datum/species/machine/handle_life(mob/living/carbon/human/H) // Handles IPC starvation
+	if(isLivingSSD(H)) // We don't want AFK people dying from this
+		return
+	if(H.nutrition >= NUTRITION_LEVEL_HYPOGLYCEMIA)
+		return
+
+	var/obj/item/organ/internal/cell/microbattery = H.get_organ_slot("heart")
```

Comment:
Also, can you reword your PR? The microbattery isn't the brain organ. This is technically heart damage